easybuild/easyconfigs/g/GDB/GDB-10.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDB'
+version = '10.1'
+
+homepage = 'https://www.gnu.org/software/gdb/gdb.html'
+description = "The GNU Project Debugger"
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('texinfo', '6.7'),
```

Comment:
Does GDB need all of `texinfo` to build? If it only requires `makeinfo` then we can build that directly instead, which is simpler than the `texinfo` install (see https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/makeinfo/makeinfo-6.7-GCCcore-9.3.0.eb).

Reply:
You are right, makeinfo is enough, thanks!